go.mod

```diff
@@ -80,6 +80,8 @@ require (
 	gopkg.in/yaml.v2 v2.4.0
 )
 
+require github.com/open-telemetry/opentelemetry-collector-contrib/extension/observer/dockerobserver v0.41.0
```

Comment:
nit, mind adding to require block above?

Reply:
Done, thanks. I assumed `go mod tidy` would have put this in place for me.